src/components/PersonLink/PersonLink.tsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import { Link, useParams, useSearchParams } from 'react-router-dom';
+import classNames from 'classnames';
+import { Person } from '../../types/Person';
+
+type Props = {
+  person: Person;
+  people: Person[];
+};
+
+const PersonLink: React.FC<Props> = ({ person, people }) => {
```

Comment:
It's not a link, this component contains other user information too, not only link